packages/terminal/src/index.ts

```diff
@@ -1,21 +1,20 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
-import { IInstanceTracker, MainAreaWidget } from '@jupyterlab/apputils';
+import { Widget } from '@phosphor/widgets';
 
 import { Token } from '@phosphor/coreutils';
 
-import { Terminal } from './widget';
-
-import '../style/index.css';
+import { IInstanceTracker, MainAreaWidget } from '@jupyterlab/apputils';
 
-export * from './widget';
```

Comment:
couldn't keep it :sob:

Reply:
That's a real shame! Do you think the situation will be improved with `esnext`?

Reply:
put it back by moving everything else (token, etc) into `constants`, so the test and example API can stay the same. this concentrates the changes to just the extension.

Reply:
back to the way it was!